gcloud/datastore/test_key.py

```diff
@@ -17,8 +17,9 @@
 
 class TestKey(unittest2.TestCase):
 
+    _DEFAULT_DATASET = 'DATASET'
+
     def setUp(self):
```

Comment:
I removed most uses of `_monkeyDatasetID` in e117ca07e56a3503445d54568a93182b6d533d56.  The class default was (is still) used as the default value passed to it, and to test for it.